src/handlers/class-processor.ts

```diff
@@ -1,26 +1,27 @@
 import reflect, { ClassReflection, PropertyReflection } from '@plumier/reflect';
-import { FunctionValueHandler } from './value-handlers/function-value-handler';
-import { ObjectLiteralValueHandler } from './value-handlers/object-literal-value-handler';
-import { EnumValueHandler } from './value-handlers/enum-value-handler';
-import { TypeValueInspector } from 'src/handlers/value-handlers/type-value-inspector';
-import { PrimitiveValueInspector } from 'src/handlers/value-handlers/primitive-value-inspector';
+import { FunctionValueInspector } from './value-inspectors/function-value-inspector';
+import { ObjectLiteralValueInspector } from './value-inspectors/object-literal-value-inspector';
+import { EnumValueInspector } from './value-inspectors/enum-value-inspector';
+import { TypeValueInspector } from './value-inspectors/type-value-inspector';
```

Comment:
1. 'Inspector' word is not accurate.
2. TypeValueInspector => ClassValueHandler